### PR TITLE
feat(plugin-copy): support info option

### PR DESCRIPTION
**Pull request by JSerFeng**
## Summary

Support object-type `info` option for plugin-copy

## Test Plan

The tests describes that after setting `info.minimized` to `true`, the file won't be processed by minimizer

### CODE REVIEW

**Comment by clijiac** [comment]
we need this PR, now the rspack will minify code in Public folder we just copy

**Comment by JSerFeng** [comment]
> we need this PR, now the rspack will minify code in Public folder we just copy

Sorry I thought this was merged, I'll make this work in the next release

File: crates/rspack_plugin_copy/src/lib.rs [hunk 1]
```diff
@@ -597,8 +622,64 @@ fn escape_glob_chars(s: &str) -> String {
...
 #[test]
 fn test_escape() {
   assert_eq!(escape_glob_chars("a/b/**/*.js"), r#"a/b/\*\*/\*.js"#);
   assert_eq!(escape_glob_chars("a/b/c"), r#"a/b/c"#);
 }
+
+// If this test fails, you should modify `set_info` function, according to your changes about AssetInfo
+// Make sure every field of AssetInfo is considered
+#[test]
+fn ensure_info_fields() {
```
Comment (ahabhgk) [inline crates/rspack_plugin_copy/src/lib.rs:670]: Why can't just use AssetInfo

Comment (JSerFeng) [reply crates/rspack_plugin_copy/src/lib.rs:670]: User specify the info config, so user can specify any field they want, for the other fields, stay untouched, so users info fields are all optional

**Review by ahabhgk — APPROVED** [review]

**Pull request merged**